components/elm/docs/tech-guide/longwave_radiation.md

```diff
@@ -0,0 +1,283 @@
+# Overview
+
+The longwave radiation in ELM solves the amount of longwave
+radiation absorbed by the ground and the vegetation, and the
+amount of outgoing radiation to the atmosphere (Figure 1).
+The model represents the ground surface as a mixture of snow,
+soil, and standing surface water. The shaded and sunlit leaves
+are combined as a single leaf within the model. The incoming
+longwave atmosphere radiation, $L^\downarrow_{atm}$, is a
+boundary condition for the model.
+
+![Image title](../figures/longwave_radiation.png)
+
+Fig 1. Two-stream longwave radiation model for
+(a) non-vegetated and (b) vegetated surfaces.
+
+## Governing Equations For Non-vegetated Surfaces
+
+The emitted longwave radation from ground, $L^{\uparrow}_g$, is
+
+$$
+\begin{equation}
+L^{\uparrow}_{g} = (1 - \epsilon_g)L^\downarrow_{atm} + \epsilon_g \sigma T_{g}^4
+\label{eq:lg_up_nonveg}
+\end{equation}
+$$
+
+where $\epsilon_g$ is the emissivity of the ground,
+$\sigma$ is the Stefan-Boltzmann constant, and
+$T_g$ is the ground temperature. The first term on the
+right-hand side represents the reflected atmospheric longwave
+radiation, while the second term represents the emitted longwave
+radiation by the ground. The emitted longwave radiation is computed as
+
+$$
+\begin{equation}
+\epsilon_g \sigma T_g^4 = \epsilon_g \sigma \left[ f_{sno} T^4_{sno} + \left( 1 - f_{sno} - f_{h2osfc}\right) T^4_{soi,1} + f_{h2osfc} T^4_{h2osfc} \right]
+\label{eqn:tg}
+\end{equation}
+$$
+
+where $T_{sno}$, $T_{soi,1}$ and $T_{h2osfc}$
+the temperature of the top snow layer, the first soil layer, and
+the standing surface water, respectively, and
+$f_{sno}$ and $f_{h2osfc}$ are fraction of snow and
+standing surface water.
+
+The radiation absorbed by the ground, $\overrightarrow{L}_g$, is
+
+$$
+\begin{equation}
+\overrightarrow{L}_g = \epsilon_g \sigma T_g^4 + \epsilon_g L_{atm}^\downarrow
+\label{eqn:lg_net_nonveg}
+\end{equation}
+$$
+
+## Governing Equations For Vegetated Surfaces
+
+The longwave radiation below the canopy, $L_v\downarrow$, is
+
+$$
+\begin{equation}
+L_v^\downarrow = (1 - \epsilon_v)L_{atm}^\downarrow + \epsilon_v \sigma T_v^4
+\end{equation}
+$$
+
+where $\epsilon_v$ is the emissivity of the vegetation
+and $T_v$ is the temperature of the canopy. The model assumes
+the sunlit and shaded leaves are at the same temperature.
+The first term on the right-hand side of the equation represents
+the transmitted atmospheric longwave radiation through the
+canopy and the second term represents the emitted longwave
+radiation by the canopy.
+
+The upwelling longwave radiation from the ground is
+
+$$
+\begin{eqnarray}
+L_g^\uparrow &=& (1 - \epsilon_g) L_v^\downarrow + \epsilon_g \sigma T_g^4 \nonumber\\[0.5em]
+&=& (1 - \epsilon_g)(1 - \epsilon_v)L_{atm}^\downarrow + (1 - \epsilon_g)\epsilon_v \sigma T_v^4 + \epsilon_g \sigma T_g^4
+\end{eqnarray}
+$$
```

Comment:
For this equation, given different surface types have different emissivity, how does ELM calculate ground emissivity? So this equation can be used to calculate ground temperature (mean of snow, soil and surface water)?

Reply:
I will add the missing equation.

Reply:
done